src/index.js

```diff
@@ -1,13 +1,42 @@
-import _ from 'lodash';
+/* eslint-disable no-return-assign */
 import './style.css';
 
-function component() {
-  const element = document.createElement('div');
+const url = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/mchCFxyMLOw3tEZP2NKf/scores/';
+const leaderboard = document.getElementById('leaderboard');
+const refresh = document.getElementById('refresh');
+const name = document.getElementById('name');
+const score = document.getElementById('score');
+const submit = document.getElementById('add');
 
-  // Lodash, imported
-  element.innerHTML = _.join(['Hello', 'webpack'], ' ');
+const createGame = (name) => {
+  leaderboard.innerHTML = '';
+  name.result.map((e, index) => leaderboard.innerHTML += `<li class="list-group-item">${name.result[index].user} ${name.result[index].score}</li>`);
 
-  return element;
-}
+  leaderboard.appendChild(name);
```

Comment:
* This line is throwing an error as appendChild is expecting a Node but instead got a string(name) please make the necessary adjustments
![image](https://user-images.githubusercontent.com/52497246/128317036-06774221-58ae-4ac4-8127-38cb1c7285cb.png)

Reply:
Well done